dash/requirements.txt

```diff
@@ -1,12 +1,33 @@
-beautifulsoup4
-dash
-dash-bootstrap-components
-dash-dangerously-set-inner-html
-dash-multi
-humanize
-frontmatter
-gunicorn
-markdown2
-openpyxl
-pandas
-pytz
+beautifulsoup4==4.11.1
```

Comment:
This is already in master. Can you merge the latest changes from master into this branch please?

Reply:
ok should be updated now.